app/views/questionnaire.py

```diff
@@ -273,6 +319,7 @@ def submit_answers(routing_path, eq_id, form_type, collection_id):
             routing_path,
         )
 
+        message = json.dumps(message)
```

Comment:
why are we doing this? does it change the structure of what we encrypt?

Reply:
See #1329

Reply:
@dwyeradam this has been changed to be done earlier : `message = json.dumps(convert_answers(`